### PR TITLE
Add instructions to README

### PR DESCRIPTION
**Pull request by dangron**
README for users and contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
-A collection of Core Magento fixes currently waiting to be merged into Magento
+A collection of Core Magento fixes currently waiting to be merged into Magento.
+
+Sometimes you encounter a core magento bug which has been fixed in the main repo but is not yet distributed, this can be
+frustrating and you might want the fix yesterday, magento2-patches collects these patches and utilises `vaimo/composer-patches`
+to apply them to your code base.
+
+## Installation
+
+`composer require space48/magento2-patches`
```
Comment (punkstar) [inline README.md:11]: What's the plan with tagging / versioning this package? Is there any point? If not then we should update these instructions to pull from `master` and ignore stability constraints in the project `composer.json`:

    composer require space48/magento2-patches:dev-master

Comment (dangron) [reply README.md:11]: I don't know to be honest, lets see what @joshdifabio thinks when he is back in on Monday.

I like that there are no tags and we live on the bleeding edge. On the other hand I don't like that it forces all the patches on you when you might just need one.

Comment (dangron) [reply README.md:11]: Master is protected and only a select few sacred folks are able to push to master right now, this gives regular troops the ability to create branches and test locally with relative ease without the risk of breaking everyones website.

Comment (joshdifabio) [reply README.md:11]: I don't see any benefit in versioning,  I think we just make sure things work before they reach master. Open to other opinions though!

**Pull request merged**